src/User/Handler.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
You can add the most useful extension sin your `package.yaml` file so you don't have to specify every time you use them:
```
default-extensions:
  - OverloadedStrings
  - QuasiQuotes
  - RecordWildCards
```